contents/founders/what-recruiters-see.md

```diff
@@ -0,0 +1,77 @@
+
+## But first, some context
+
+I'm talking about startups with a reasonable degree of brand recognition (ie. not a startup founded last week). In PostHog's case, we are:
+
+- 42 people, $27m raised
+- Team is 70% engineers
+- 1 in-house recruiter, no agencies
+
+Let’s get into it. 
+
+## So what happens to my application after I hit send?
+
+If you're applying for a job at a startup, chances are they are using an applicant tracking system (ATS) like Ashby, Greenhouse, Lever, or Workable. We use Ashby, but the process a recruiter follows generally doesn't vary massively between these systems. This is what a typical application looks like:
+
+![Ashby screenshot](../images/ashby-screenshot.png)
+
+## How do I maximize my chances of getting through?
+
+This stuff is subjective, but generally these are the things that can increase the chances of a good application making it through:
+
+- Write a short and actually personalized proper cover note
+  - A few sentences about why you are interested in this role specifically is fine
+  - You don't need 5 paragraphs that are just your resumé slightly reworded - it's just more text we will skim over
+  - Our ATS will wreck your formatting beyond line breaks
+  - Don't introduce yourself (we know who you are)
+  - Match the tone of the company - for example, we are informal so please no 'Dear Sir/Madam'
+  - Write _something_ - if it's too much hassle, that's ok, you're probably not that interested!
+  - No ChatGPT - they're really obvious
+- No profile pictures - this isn't social media, and you're wasting the space
+- Stick to boring formatted CVs - prioritize clarity over design
+  - I'd go so far as to say that exporting your LinkedIn profile as a PDF is totally fine, but...
+- Keep your LinkedIn up to date - it makes us suspicious when there are big discrepancies between the two
+- Don't have your CV end in 2021 or whenever, we don't have time to contact you to query it - we don't care if you had a career break, just put that
+- Emailing the recruiter/hiring manager directly causes more stress because we then have to add you to our ATS - please apply directly, we are dying here
+
+## I did all this and it didn't work - where's my feedback, you assholes?
+
+When you apply for a job at a startup and don't get past the first stage, you'll either get nothing back (booo) or an automated email saying you have been unsuccessful. Applicants who felt they were a great fit might, quite reasonably, ask for feedback as to why they weren't successful. 
+
+We find ~5% of candidates ask for feedback - if we replied to each one and spent ~5min writing each email, we'd be spending 30-60min on a typical day sending people feedback. This is why we tell people to only expect feedback after any interview stage. I expect this picture is similar for most startups.
+
```

Comment:
I often feel bad for those that look well qualified but we don't hire. Maybe explain why this can happen? The point being, don't feel like it's necessarily a reflection of you - it's a reflection of your relative competition... you may have been competing against someone that has worked with us for 5 years before

Reply:
Agree, added something on that now